core/field_label.ts

```diff
@@ -66,9 +65,9 @@ export class FieldLabel extends Field {
     this.setValue(opt_value);
   }
 
-  override configure_(config: AnyDuringMigration) {
+  override configure_(config: Config) {
```

Comment:
I notice that `configure_` is protected in some classes, like `core/field_image.ts`, and not in others like here. Are these distinctions on purpose? (It looks like it's how the original js classes were, though perhaps it was a pre-existing problem)

If it's on purpose and `configure_` is intended to be called for this class, then line 70 has a minor functionality difference where including `class` on the first call of `configure_(...)` then removing `class` from the object in the follow-up call, the second call will not properly reset `this.class_` to `null` (or `undefined` by the previous functionality)

Reply:
Fixed the visibility modifiers. Left the implementation the same because multiple calls are not supported.